### DISCUSSION TITLE
Images rotated?

### DISCUSSION THREAD

**Discussion by bluedogtwo** [pdfcpu/pdfcpu#911] [Q&A]
I used the following command to make a pdf page from three images, but the images got rotated. What can I do?
pdfcpu grid -- "bo:off" out.pdf 3 1 l1.png l2.png l3.png

**Comment by hhrutter** [comment]
Hello!

I need more info about this.
Do you also get rotated images when you just `pdfcpu import ` them?
Maybe you can post a screenshot.

**Comment by bluedogtwo** [comment]
With import it is not rotated.

screenshot
![img](https://github.com/pdfcpu/pdfcpu/assets/14124009/de398a85-847a-4448-a57a-c914bc94c951)

**Comment by hhrutter** [comment]
Can you send me both result pdfs (from grid and import) for analysis?
Otherwise I have to guess what I am looking at.

**Comment by bluedogtwo** [comment]
[aaa.pdf](https://github.com/user-attachments/files/16164331/aaa.pdf)
[abc.pdf](https://github.com/user-attachments/files/16164332/abc.pdf)

**Comment by hhrutter** [comment]
Have you tried switching the grid dimensions?

`pdfcpu grid -- "bo:off" out.pdf 1 3 l1.png l2.png l3.png`
instead of
`pdfcpu grid -- "bo:off" out.pdf 3 1 l1.png l2.png l3.png`

Also:
aaa.pdf is the result of ?
abc.pdf is the result of ?

**Comment by bluedogtwo** [comment]
Using
pdfcpu grid -- "bo:off" out.pdf 1 3 l1.png l2.png l3.png

[13.pdf](https://github.com/user-attachments/files/16416874/13.pdf)

aaa.pdf is result of
pdfcpu import aaa.pdf aaa.png

abc.pdf is the result of
pdfcpu grid -- "bo:off" abc.pdf 3 1 l1.png l2.png l3.png

**Comment by hhrutter** [comment]
`nup` and `grid` are enforcing a best-fit orientation of individual content onto a page when rendering.

If you get the latest commit, then you can use `enforce:off` to achieve what you want which is keep the original orientation of the content.

**Comment by hhrutter** [comment]
Closed due to inactivity.
